integration/spark/app/src/main/java/io/openlineage/spark/agent/ArgumentParser.java

```diff
@@ -126,6 +131,11 @@ private static void extractSparkSpecificConfigEntriesFromSparkConf(
     findSparkConfigKey(conf, SPARK_CONF_PARENT_JOB_NAMESPACE)
         .ifPresent(config::setParentJobNamespace);
     findSparkConfigKey(conf, SPARK_CONF_PARENT_RUN_ID).ifPresent(config::setParentRunId);
+    findSparkConfigKey(conf, SPARK_CONF_ROOT_PARENT_JOB_NAMESPACE)
```

Comment:
why do we need this? Shouldn't this be covered automatically by jackson? `findSparkConfigKey` was mainly used to cover legacy settings and shouldn't be added for newly introduced conf entries.

Reply:
You're right, removed.